fuzz-all.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+fuzz_list_cmd="cargo fuzz list"
```

Comment:
Please move this script to the *fuzz* directory. Also please fix all trailing whitespaces.